src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int OPERATION_INDEX = 0;
+    public static final int COUNT_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        File file = new File(fromFileName);
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/186673256-f3d8ff33-8c82-472d-bf8b-7b394e027c73.png)
Before you create PR with your homework, you need to go through the checklist under the task and correct all the points described there. The mentor will not check the work until the checklist points are corrected” and stop homework review